src/amo/components/PermissionsCard/permissions.js

```diff
@@ -68,28 +63,18 @@ export class PermissionUtils {
 
   // Get lists of optional and required permissions from the correct platform file.
   getCurrentPermissions({
-    platformFiles,
-    userAgentInfo,
-    _findFileForPlatform = findFileForPlatform,
+    file,
   }: GetCurrentPermissionsParams): {
     optional: Array<string>,
     required: Array<string>,
   } {
-    const file = _findFileForPlatform({
-      userAgentInfo,
-      platformFiles,
-    });
     const permissions = {
       optional: [],
       required: [],
     };
 
     if (!file) {
```

Comment:
is `file` being falsey still a condition we need to account for?  (wouldn't it be better that the caller of `getCurrentPermissions` didn't call it when `file ` was `undefined`?)

Reply:
Current caller all check for this, but `file` is defined as `AddonFileType`, which can be `null` because of how it's set, so I believe it's best to keep this until we can refactor the API to replace `files` list with a single `file` object that should always be present (and not `null`)